MiniVibe/MiniVibe/Models/NowPlaying.swift

```diff
@@ -18,15 +18,48 @@ class NowPlaying: ObservableObject {
     }
     
     func addTrack(track: TrackInfo) {
+        isPlaying = true
         if let index = upNext.firstIndex(of: track) {
             upNext.insert(upNext.remove(at: index), at: 0)
         } else {
             upNext.insert(track, at: 0)
         }
     }
     
+    func deleteTrack() {
+        selectedTracks.forEach { upNext.remove(at: upNext.firstIndex(of: $0) ?? -1) }
```

Comment:
`upNext.firstIndex(of: $0)` 가 존재하지 않을 때 기본값으로 `-1`을 주기보단, 옵셔널 바인딩으로 좀 더 안전하게 처리해주는게 더 좋을 것 같아요.
```suggestion
        selectedTracks.forEach { 
            guard let selectedIndex = upNext.firstIndex(of: $0) else { return }
            upNext.remove(at: selectedIndex) 
        }
```

Reply:
그러네요! -1 처럼 무의미한 값이져.
안전하게 풀고 커밋할게요?